MusicCat-Console/Program.cs

```diff
@@ -52,5 +64,40 @@ static void Main(string[] args)
 			logStream?.Dispose();
             Listener.Stop();
         }
+
+		private static int counter;
+		private static void OnConfigChange(object sender, FileSystemEventArgs e)
+		{
+			if (counter == 0)
+			{
+				counter++;
+				return;
+			}
+
+			counter = 0;
+			try
+			{
+				logWriter?.Dispose();
+				logStream?.Dispose();
+				Listener.Config = Config.ParseConfig(out logStream, out logWriter);
+			}
+			catch (Exception ex)
+			{
+				Log(new ApiLogMessage($"Failed to de-serialize config, using default config instead. Exception: {ex.Message}{Environment.NewLine}{ex.StackTrace}", ApiLogLevel.Warning));
+				Listener.Config = Config.DefaultConfig;
+			}
```

Comment:
The correct behavior on a failing config change should be to use the _current_ config, not the default config.